### PR TITLE
NEWS.md: interim update

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -31,13 +31,76 @@ The format is based on [Keep a Changelog]
...
+
+- Support pipe menus. See labwc-menu(5) for usage.
+- Add snap-to-edge overlay. Written-by: @tokyo4j. PR #1652 #1702
+  This includes the following new settings:
+
+```
+snapping.preview.[region|edge].fill: yes|no
+snapping.preview.[region|edge].bg.color: #8080b380
+snapping.preview.[region|edge].border.color: #ffffff,#000000,#ffffff
+snapping.preview.[region|edge].border.width: 1
```
Comment (Consolatis) [inline NEWS.md:93]: This needs to be revisited once
- #1737

is merged.

File: NEWS.md [hunk 2]
```diff
@@ -102,6 +165,25 @@ contributors as noted in the log.
 
 ### Fixed
 
+- Notify idle manager when emulating cursor move (forgotten in original
+  implementation).
```
Comment (Consolatis) [inline NEWS.md:169]: I don't think we need the forgotten part here.

File: NEWS.md [hunk 3]
```diff
@@ -102,6 +165,25 @@ contributors as noted in the log.
...
+- Fix GrowToEdge/ShrinkToEdge action bug caused by clients ignoring the
+  requested size, for example a terminal honouring size-hints.
+- Fix `assert()` on VT switch. Issue #1667
+- Use pre-multiplied colors by default to ensure consistent colors for
+  titlebars with transparency (because the corner pieces uses different
+  technology to the remainder of the titlebar). #1684
```
Comment (Consolatis) [inline NEWS.md:175]: This feels like too much implementation details, suggest something like "Ensure the titlebar has a consistent look when using transparency".

File: NEWS.md [hunk 4]
```diff
@@ -102,6 +165,25 @@ contributors as noted in the log.
...
+- Fix dnd bug where dnd does not finish properly on cursor-button-release
+  if there is no surface under the cursor such as on the desktop when no
+  background client is running. #1673
+- Send cursor-button release event to CSD client before finishing window
+  dragging to avoid a bug whereby the release event is incorrectly sent a
+  layer-shell client at the end of a drag.
```
Comment (Consolatis) [inline NEWS.md:181]: `sent *to* a layer-shell client`

File: NEWS.md [hunk 5]
```diff
@@ -137,6 +219,8 @@ contributors as noted in the log.
 
 ### Changed
 
+- In theme settings, mark color definitions in the format #rrggbb aaa` as
+  deprecated (still supported, but will removed in some future release).
```
Comment (Consolatis) [inline NEWS.md:223]: Suggest adding something like "in favor of the more commonly used `#rrggbbaa` format"

**Comment by johanmalm** [comment]
Thanks for review. All makes sense. Will update over the next few days.

**Comment by johanmalm** [comment]
All previous comments addressed. 
Ready for review again.
Need squash before merge.

**Pull request merged**